scripts/build/build-docs.js

```diff
@@ -38,12 +39,13 @@ shell.cp(`${prettierPath}/index.js`, `${docs}/index.js`);
 shell.exec(
   `node_modules/babel-cli/bin/babel.js ${docs}/index.js --out-file ${docs}/index.js --presets=es2015`
 );
-shell.sed(
-  "-i",
-  /(var crypto=true;)/,
-  "try{$1}catch(e){}",
-  `${docs}/index.js`
-);
+
+// wrap content with IIFE to avoid `assign to readonly` error on Safari
+(function(filename) {
```

Comment:
I think you can just set `format` to `iife` in `rollup.docs.config.js`?

Reply:
We built `lib/index.js` from `rollup.index.config.js`, and we need to adjust a lot to get it worked, IMO this is the easiest fix.